src/apps/fuzzers/libfuzzer.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2021 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdint.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "h3api.h"
+#include "utility.h"
+
+int
+LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+    char *new_str = (char *)malloc(size+1);
+    if (new_str == NULL){
+        return 0;
+    }
+    memcpy(new_str, data, size);
+    new_str[size] = '\0';
+
+    
+    H3Index h3;
+    h3 = H3_EXPORT(stringToH3)(new_str);
```

Comment:
I'm a little unclear what's happening here. It looks like the fuzzer program is providing a block of input data, and this is being converted into H3 indexes? Would it make sense to just read 8 bytes (64 bits) out of that input data and treat it as an H3 Index, rather than going through stringToH3? Or is the intent to include stringToH3 in the fuzzing?

Nit that stringToH3 seems to be the only one using H3_EXPORT - I'd just remove usage of that from this file.

Reply:
The intention was to both fuzz `stringToH3` as well as create a new `H3Index`.